aiohasupervisor/root.py

```diff
@@ -109,12 +109,19 @@ async def info(self) -> RootInfo:
         result = await self._client.get("info")
         return RootInfo.from_dict(result.data)
 
+    async def reload_updates(self) -> None:
+        """Reload updates.
+
+        Reload main components update information (OS, Supervisor, Core and plug-ins).
+        """
+        await self._client.post("reload_updates", timeout=ClientTimeout(total=300))
+
     async def refresh_updates(self) -> None:
-        """Refresh updates."""
+        """Refresh updates (discouraged)."""
         await self._client.post("refresh_updates", timeout=ClientTimeout(total=300))
 
     async def available_updates(self) -> list[AvailableUpdate]:
-        """Get available updates."""
+        """Get available updates (discouraged)."""
```

Comment:
Is this discouraged as well? How do you get the list of updates then?

Reply:
The Core isn't using this API to learn about updates, it just checks the `xy/info` endpoint of the individual components. Hence I don't really see a point of maintaining it :shrug: 

That said, I just realized that the CLI uses that API as well, so maybe we should keep it for that :thinking:

Reply:
I chose to keep `available_updates` as non-discouraged for now. It should be fairly easy to maintain on Supervisor side and is really quite nice to have for the CLI.